portfolio/src/main/webapp/style.css

```diff
@@ -84,30 +81,32 @@ body{
 .cardTitle{
   font-size: 20px;
   margin: 0px;
-  margin-bottom:10px
+  margin-bottom: 0.1rem;
 }
 
 
 .cardSecondary{
+  margin: 0px;
 }
 
 .cardLongDesc{
   display: none;
   opacity: 0.0;
+  border-top: #E0E0E0 1px solid;
+  margin-top: 0.4rem;
+  padding-top: 0.4rem;
 }
 
 .openCard{
-  /*min-height: 300px;*/
```

Comment:
Try to actually delete unused code when you end up committing. It helps keep the codebase clean.